components/file-upload/src/HeadLessFileUpload.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ForwardedRef, forwardRef } from "react";
+
+export interface HeadLessFileUploadProps {
+	acceptedFileTypes?: Array<string>;
+	allowsMultiple?: boolean;
+	handleAfterFileUploadHook?: (files: FileList | null) => Promise<void>;
+	children?: React.ReactNode;
+	className?: string;
+}
+
+function HeadLessFileUpload(
+	props: HeadLessFileUploadProps,
+	ref: ForwardedRef<HTMLInputElement>,
+) {
+	const {
+		handleAfterFileUploadHook,
+		acceptedFileTypes,
+		allowsMultiple,
```

Comment:
Can pass default values here: 
```suggestion
		allowsMultiple = false,
```

Reply:
passed